crates/sui-adapter-transactional-tests/tests/child_count/non_zero_child_count_valid.exp

```diff
@@ -21,21 +21,21 @@ created: object(112), object(113), object(114)
 written: object(111)
 
 task 5 'run'. lines 58-58:
-written: object(113), object(115)
+Error: Object 0x4ed62609e80f1a882d16311822ec560fbc1d81a2ab0bcb4fd6358910dee9573f is owned by object 0x0e1963f04364076254e2239b46ad24cccf0ce9c4adec48c942e5180ff59ed753. Objects owned by other objects cannot be used as input arguments.
```

Comment:
@tnowacki, so is this expected now for the behavior to differ vs before that PR?